src/main/resources/db/changeLog.xml

```diff
@@ -42,4 +42,12 @@
             <column name="modifiedat" type="datetime(6)"/>
         </createTable>
     </changeSet>
+
+    <changeSet id="project_status" author="daniil smirnov">
+        <addColumn tableName="project">
+            <column name="status" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
```

Comment:
We should have an SQL script to update all the current statuses to some initial one

Reply:
```suggestion
        </addColumn>
        <sql>UPDATE project SET status ='INITIATION'</sql>
```